daemon/src/maker_cfd_actor.rs

```diff
@@ -207,33 +215,23 @@ impl MakerCfdActor {
         };
 
         // 2. Insert CFD in DB
-        // TODO: Don't auto-accept, present to user in UI instead
         let cfd = Cfd::new(
             current_order.clone(),
-            quantity,
-            CfdState::Accepted {
+            msg.quantity,
+            CfdState::IncomingOrderRequest {
                 common: CfdStateCommon {
                     transition_timestamp: SystemTime::now(),
                 },
+                taker_id,
             },
         );
         insert_cfd(cfd, &mut conn).await?;
 
-        self.takers()?
-            .do_send_async(maker_inc_connections_actor::TakerMessage {
-                taker_id,
-                command: TakerCommand::NotifyOrderAccepted { id: order_id },
-            })
-            .await?;
         self.cfd_feed_actor_inbox
             .send(load_all_cfds(&mut conn).await?)?;
 
         // 3. Remove current order
         self.current_order_id = None;
-        self.takers()?
-            .do_send_async(maker_inc_connections_actor::BroadcastOrder(None))
-            .await?;
-        self.order_feed_sender.send(None)?;
```

Comment:
I think these have to stay as well, otherwise the UI and all other takers will be out of sync.